src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -63,11 +86,123 @@ const NavMenu = (): ReactElement => {
-              <MenuItem component="a">
-                <ListItemText primary={t('Reports')} />
-              </MenuItem>
-            </HandoffLink>
+            <MenuItem
+              ref={anchorRef}
+              aria-controls={reportsMenuOpen ? 'menu-list-grow' : undefined}
+              aria-haspopup="true"
+              onClick={handleReportsMenuToggle}
+            >
+              <ListItemText primary={t('Reports')} />
+            </MenuItem>
+            <Popper
+              open={reportsMenuOpen}
+              anchorEl={anchorRef.current}
+              role={undefined}
+              transition
+              disablePortal
+            >
+              {({ TransitionProps, placement }) => (
+                <Grow
+                  {...TransitionProps}
+                  style={{
+                    transformOrigin:
+                      placement === 'bottom' ? 'center top' : 'center bottom',
+                  }}
+                >
+                  <Paper>
+                    <ClickAwayListener onClickAway={handleReportsMenuClose}>
+                      <MenuList
+                        autoFocusItem={reportsMenuOpen}
+                        id="menu-list-grow"
+                      >
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/donations"
+                          as={`/accountLists/${state.accountListId}/reports/donations`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText primary={t('Donations')} />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/partner"
+                          as={`/accountLists/${state.accountListId}/reports/partner`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Partner)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/salary"
+                          as={`/accountLists/${state.accountListId}/reports/salary`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Salary)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/designation_accounts"
```

Comment:
```suggestion
                          href="/accountLists/[accountListId]/reports/designationAccounts"
```
We can choose a different naming convention (I might suggest `kebab-case) but someone started with `accountLists` being camel case.

Reply:
I'm fine renaming the file names and the paths.  I was carrying over the names from the existing site, but I agree that many of them have strange formatting or are not descriptive enough of what the report is about.